Wordle.Api/Wordle.Api/Services/WordOfTheDayService.cs

```diff
@@ -60,6 +61,25 @@ public async Task<string> GetWordOfTheDay(DateOnly date)
         return wordOfTheDay.Word!.Text;
     }
 
+    //Attempted Api Service for getting stats
+    /* 
+    public async Task<GameStatsDto> GetWordStats(string word)
+    {
+      var wordStats = await Db.Words
+        .Where(w => w.Text.Equals(word, StringComparison.OrdinalIgnoreCase))
+        .Select(w => new GameStatsDto
+        {
+          Word = w.Text,
+          AverageGuesses = w.AverageGuesses,
+          TotalTimesPlayed = w.TotalTimesPlayed,
+          TotalWins = w.TotalWins,
+          TotalLosses = w.TotalLosses
+        })
```

Comment:
this can be implemented in gameservice.cs rather than this file

Reply:
Refactored, thank you!